Player.java

```diff
@@ -1,37 +1,24 @@
-import java.util.*;
-import java.io.*;
-
-public class Player{
-   private String name;
-   private int space;
+public class Player
+{
+   private final String name;
+   private int currentSpace;
    
-   public Player(String n){
-      space = 0;
-      name = n;
-   }
+   public Player(String n)
+   { currentSpace = 0; name = n; }
    
    public void setSpace(int s){

```

Comment:
same naming problem here. the parameter should b called @ least `newSpace` or something.